src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.test.ts

```diff
@@ -1,224 +0,0 @@
-/*
```

Comment:
Moved to `src/plugins/unified_histogram/public/services/lens_vis_service.suggestions.test.ts`